### PR TITLE
Add show_title checkbox for contentpage

### PR DESCRIPTION
**Pull request by raphael-s**
Adds a checkbox for contentpages which allows a user to decide whether the title of the contentpage is displayed or not.
This works with the new `IContentPageShowTitle` behavior.

See: https://github.com/4teamwork/winterthur.web/issues/312

### CODE REVIEW

File: ftw/simplelayout/contenttypes/profiles/default/types/ftw.simplelayout.ContentPage.xml [hunk 1]
```diff
@@ -33,6 +33,7 @@
     <property name="behaviors">
         <element value="ftw.simplelayout.interfaces.ISimplelayout" />
         <element value="ftw.simplelayout.opengraph.interfaces.IOpenGraphSupport" />
+        <element value="ftw.simplelayout.interfaces.IContentPageShowTitle" />
```
Comment (jone) [inline ftw/simplelayout/contenttypes/profiles/default/types/ftw.simplelayout.ContentPage.xml]: @raphael-s We don't need this behavior by default. When needed, we can activate it in the customer project.

So I propose to:

- [ ] Do not enable it in `ftw.simplelayout.ContentPage.xml`
- [ ] Make sure the changes in `simplelayout.pt` do not break when the behavior is not available
- [ ] Document the existence of the behavior in the Readme.

**Comment by raphael-s** [comment]
@Tschanzt added a few tests, I think this should be ready for a merge now :)

File: ftw/simplelayout/browser/simplelayout.py [hunk 2]
```diff
@@ -7,7 +7,8 @@
 from zope.publisher.browser import BrowserView
 import json
 import logging
-
+from ftw.simplelayout.interfaces import IContentPageShowTitle
+from zope.component import queryAdapter
```
Comment (Tschanzt) [inline ftw/simplelayout/browser/simplelayout.py]: ✂️

**Comment by Tschanzt** [comment]
👍

**Pull request merged**